src/views/SignTransactionLedger.vue

```diff
@@ -153,8 +167,23 @@ export default class SignTransactionLedger extends Vue {
                 label: this.rpcState.origin.split('://')[1],
                 image: checkoutRequest.shopLogoUrl,
             };
+        } else if (this.request.kind === RequestType.CASHLINK) {
```

Comment:
I guess these `if`s can now be a `switch`.

Reply:
I wouldn't do it now to avoid unnecessary merge conflicts with the checkout/ledger changes as using a switch instead of if/else changes the indentation of the code.
For this reason I also don't necessarily prefer a `switch` here.

Reply:
I would prefer a `switch` here. If it leads to merge conflicts, it can be done in the next PR touching this though.